examples/lines_bars_and_markers/categorical_variables.py

```diff
@@ -0,0 +1,39 @@
+"""
+==============================
+Plotting categorical variables
+==============================
+
+How to use categorical variables in matplotlib.
+
+Many times you want to create a plot that uses categorical variables
+in Matplotlib. For example, your data may naturally fall into
+several "bins" and you're interested in summarizing the data per
+bin. Matplotlib allows you to pass categorical variables directly to
```

Comment:
Don't think you need this sentence since this is a histogram:
>For example, your data may naturally fall into several "bins" and you're interested in summarizing the data per bin.